### ISSUE TITLE
Support for gothic 2 mods?

### ISSUE THREAD

**Issue by i30817** [REGoth-project/REGoth#363]
I was thinking of trying Velaya with this because my computer is kind of crap and shutdowns on overheating unless i place the GPU to the lowest performance.

Didn't see instructions to load mods here, i normally use GothicStarter and wine.

Is there any support for launch mods planned?

**Comment by frabert** [comment]
You can use the `-m modfile.mod`option to launch mods

**Comment by i30817** [comment]
Interestingly, Velaya installs have two .mod files, neither of which work but one at least changes the menu image.

They still try to load the main game (Xardas tower with the nameless hero) instead of the mod start.
Also REGoth doesn't seem prepared for gaming gothic 2, it's slower than the original and you have no collision (besides being unable to change or use the original shortcuts).

I'm still interested in the project but i'm likely going to wait a year or two before retrying. If i know something about open source engine re implementations is that they take a long while - especially if  they're done on a complex language like C++ - even exult and scummvm aren't perfect yet (exult because the codebase is a mess and few coders, scummvm because it's just that complicated).

**Comment by frabert** [comment]
I believe you can use the `-m` switch more than once.

Yes, the project is still under heavy development and the team is small, the performance under windows is not stellar, but you can get some more FPS out of it by using a Release build instead of a Debug one

**Comment by i30817** [comment]
I'm using linux. Honestly i think a project like this has a lot of 'growing up'  to do and the code might become unrecognizable if it's not moribund (like say, exult) 5 years from now. 

I'm not a historian but i recall that openmw changed approaches 1 or 2 times until they decided on a scenegraph visitor approach and became much more stable and performant - I don't actually know how REGoth is architected but i can see something similar happening here if the project attracts enough talent; open source projects tend to reinvent themselves often because they can't use middleware like unity.

**Comment by frabert** [comment]
Yes, we are aware that this is _not_ an easy task. If it were, someone would have done it already :) Personally, I don't mind if somewhere down the road we'll have to rewrite something from scratch -- such is life, hindsight only works one way unfortunately. The links are very interesting, it'll be good to have some other projects to look up to
